lib/src/components/NxTextInput/types.ts

```diff
@@ -43,12 +43,15 @@ export type Props = Omit<StateProps, 'trimmedValue'> & HTMLProps & {
   onKeyPress?: ((keyCode: string) => void) | null;
   validatable?: boolean | null;
 
-  // For internal use only, this prop allows additional content to be inserted just before the
+  // For internal use only:
+  // This prop allows additional content to be inserted just before the
   // <input>. This is used by NxFilterInput
   prefixContent?: ReactNode | null;
+  // This prop sets the input attribute type to date.
+  isDateInput?: boolean;
```

Comment:
This is more what I had in mind: https://github.com/sonatype/sonatype-react-shared-components/pull/564